syntax = "proto2";

import "memory-access.proto";
import "alloc-record.proto";

message KernelInvocation {
    repeated proto.MemoryAccess accesses = 1;
    repeated proto.AllocRecord allocations = 2;
    required double kernelTime = 3;
    required string kernel = 4;
}
